source/Calamari.Common/Features/Packages/NuGet/LocalNuGetPackage.cs

```diff
@@ -43,6 +43,9 @@ static ManifestMetadata ReadMetadata(string filePath)
             {
                 foreach (var entry in archive.Entries)
                 {
+                    if (entry.Key == null)
```

Comment:
Similar concern with`NupkgExtractor`

Reply:
In this class, we're only looking for `entry.Key`s that end with `.nuspec`, so this one is okay.